src/test/ui/lint/outer-forbid.stderr

```diff
@@ -7,23 +7,5 @@ error[E0453]: allow(unused) overruled by outer forbid(unused)
 19 | #[allow(unused, unused_variables, bad_style)]
    |         ^^^^^^ overruled by previous forbid
 
-error[E0453]: allow(unused_variables) overruled by outer forbid(unused)
-  --> $DIR/outer-forbid.rs:19:17
-   |
-17 | #![forbid(unused, non_snake_case)]
-   |           ------ `forbid` level set here
-18 | 
-19 | #[allow(unused, unused_variables, bad_style)]
-   |                 ^^^^^^^^^^^^^^^^ overruled by previous forbid
-
-error[E0453]: allow(bad_style) overruled by outer forbid(non_snake_case)
-  --> $DIR/outer-forbid.rs:19:35
-   |
-17 | #![forbid(unused, non_snake_case)]
-   |                   -------------- `forbid` level set here
-18 | 
-19 | #[allow(unused, unused_variables, bad_style)]
-   |                                   ^^^^^^^^^ overruled by previous forbid
-
-error: aborting due to 3 previous errors
```

Comment:
This looks like a regression - the "overruled by outer forbid" is reported only for one lint. (But this may be intended?)